components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/IdentityMgtConfig.java

```diff
@@ -528,7 +537,7 @@ private void loadPolicyExtensions(Properties properties, String extensionType) {
 
                 this.policyRegistry.addPolicy((PolicyEnforcer) policy);
                 count++;
-
+                size--;
```

Comment:
There was a bug and the fix is provided in this PR : https://github.com/wso2/carbon-identity/pull/937

"count" is used to track the extension policy number. eg (policy extensions are 2, 3), 
Password.policy.extensions.2.pattern=^((?=.*\\d)(?=.*[a-z])(?=.*[A-Z])).{9,100}$
Password.policy.extensions.3.pattern=^((?=.*\\d)(?=.*[a-t])(?=.*[A-Y])).{9,100}$

and the "size" is used to track the number of policy extensions (number of extensions 2).